version/version.go

```diff
@@ -20,7 +20,7 @@ const (
 	// Must be a string because scripts like dist.sh read this file.
 	// XXX: Don't change the name of this variable or you will break
 	// automation :)
-	TMCoreSemVer = "0.32.0"
+	TMCoreSemVer = "0.32.1"
 
 	// ABCISemVer is the semantic version of the ABCI library
 	ABCISemVer  = "0.16.0"
```

Comment:
@thanethomson @ebuchman should we bump the version on the ABCI

Reply:
Very good question. I don't think we've ever really specified what this version refers to. Probably it should be the protobuf files themselves, and any changes to the Go wrapper should be covered by the TMCoreSemVer. In that case, since we added the CheckTxType, this should probably change to `0.16.1`. My understanding is any implementations of v0.16.0 will just ignore the new field in RequestCheckTx.

Reply:
is TMCoreSemVar suppose to be changed or the automation is supposed to do it?